osgpkitools/incommon_request.py

```diff
@@ -282,9 +285,15 @@ def main():
         config_parser = ConfigParser.ConfigParser()
         config_parser.readfp(StringIO(CONFIG_TEXT))
         CONFIG = dict(config_parser.items('InCommon'))
+        
+        if args.orgcode:
+            codes = [code.strip() for code in args.orgcode.split(',')]
+            CONFIG['organization'] = codes[0]
+            CONFIG['department'] = codes[1]
+            print("Switching organization code to %s and department code to %s" % (CONFIG['organization'], CONFIG['department']))
```

Comment:
Rather than say "Switching  ... to ... to ..." I suggest "Using ... of ... of ...".   Also might as well always say that, not just when there's an option set.

Reply:
Fixed